sqlbrite-lint/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'kotlin'
+
+dependencies {
+  compileOnly rootProject.ext.kotlinStdLib
+  compileOnly rootProject.ext.lintApi
+  compileOnly rootProject.ext.lintChecks
```

Comment:
@jrodbx We need both of these? Also is `compileOnly` correct for both?

Reply:
At least in the official custom lint rule sample, they have it like that.

Reply:
compileOnly is correct for both, since the runtime will provide the classes; however, good catch on the unnecessary `lintChecks` dependency.  that's only needed when you're referencing/subclassing from the standard lint checks.  

@geralt-encore it doesn't appear that you're relying on the standard lint checks, so you can remove the `lintChecks` dependency.

Reply:
@jrodbx thanks for the explanation! Wasn't sure about this dependency